Doc/reference/expressions.rst

```diff
@@ -740,7 +740,8 @@ which are used to control the execution of a generator function.
    because there is no yield expression that could receive the value.
 
 
-.. coroutinemethod:: agen.athrow(type[, value[, traceback]])
+.. coroutinemethod:: agen.athrow(value)
+                     agen.athrow(type[, value[, traceback]])
```

Comment:
It seems that there was an update to the generator.throw entry. Could some of that wording apply here?

https://github.com/python/cpython/commit/8be7c2bc5ad5e295f0f855bb31db412eef2c7c92#diff-c7ab338b9f88f5c444cde36aa48132cf5534500c6969a3fd2c7680234e87afd6R573

Reply:
On second thought there's probably no reason to repeat it.